kratos/utilities/force_and_torque_utils.cpp

```diff
@@ -0,0 +1,75 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+
+/* System includes */
+
+/* External includes */
+
+/* Project includes */
+#include "utilities/force_and_torque_utils.h"
+#include "utilities/parallel_utilities.h"
+#include "utilities/reduction_utilities.h"
+#include "utilities/math_utils.h"
+
+namespace Kratos
+{
+
+
+std::array<array_1d<double,3>,2> ForceAndTorqueUtils::SumForceAndTorque(const ModelPart& rModelPart,
+                                                                        const array_1d<double,3>& rReferencePoint)
+{
+    KRATOS_TRY
+
+    using CompoundArray = array_1d<double,6>;
```

Comment:
Why creating this return type instead of the `array_1<array_1d<double,3>,2>`? I guess it is to use the `SumReduction`. Note that you can define your custom one to reduce `array_1<array_1d<double,3>,2>`.